elements/lisk-transactions/src/0_transfer_transaction.ts

```diff
@@ -104,18 +101,6 @@ export class TransferTransaction extends BaseTransaction {
 			);
 		}
 
-		if (!this.fee.eq(TRANSFER_FEE)) {
```

Comment:
why we remove this constraint from this place?

Reply:
it's checked in the Bese transaction